### DISCUSSION TITLE
Unused Images Docker Desktop

### DISCUSSION THREAD

**Discussion by herculespt** [immich-app/immich#9726] [General]
Hi,

Do you know if I can delete Unused (dangling) images from Docker Desktop ? (Windows 10)
![unused](https://github.com/immich-app/immich/assets/26182991/b846e037-0d5c-410f-93b2-2e65f53bf743)

**Comment by schuhbacca** [comment]
This isn't really an immich question, more docker. But yes, if the image isn't currently being used in your instance it's more than likely safe to delete.

**Comment by herculespt** [comment]
Done
